### DISCUSSION TITLE
Spatial Bitrate Allocation in VVenC

### DISCUSSION THREAD

**Discussion by fatima0407** [fraunhoferhhi/vvenc#405] [Q&A]
Hello,

I am working on spatial bit allocation by modifying QP values for specific CTUs in video encoding using VVenC. Does VVenC support setting different QP values for different regions of an image directly through its existing options without modifying the source code? If not, are there any recommended workflows or scripts that can be used to achieve this effect?

Thank you for your assistance.

**Comment by adamjw24** [comment]
Yes, VVenC does support spatially varying QP allocation through `--qpa 1`/`--PerceptQPA=1` in vvencapp (default)/vvencFFapp (non-default).
